spec/fixtures/configs/local-bolt-server.conf

```diff
@@ -1,6 +1,10 @@
 bolt-server: {
+    host: 0.0.0.0
     port: 6789
     ssl-cert: "spec/fixtures/ssl/cert.pem"
     ssl-key: "spec/fixtures/ssl/key.pem"
     ssl-ca-cert: "spec/fixtures/ssl/ca.pem"    
+
+    loglevel: info
```

Comment:
Unrelated to this PR, but it seems like we don't actually support 'verbose' as the docs state: https://puppet.com/docs/bolt/0.x/bolt_configuration_options.html#log-file-configuration-options but support `Logging.init :debug, :info, :notice, :warn, :error, :fatal, :any`.